client/asset/btc/btc.go

```diff
@@ -388,9 +388,6 @@ type BTCCloneCFG struct {
 	ManualMedianTime bool
 	// OmitRPCOptionsArg is for clones that don't take an options argument.
 	OmitRPCOptionsArg bool
-	// LegacySendToAddr sents legacy raw tx which does not have positional fee
-	// rate param.
-	LegacySendToAddr bool
```

Comment:
That reminds me, I think there was a comment in the dcr code that says something like "TODO: just use sendtoaddress?".  Haha, guess that's out of the question now.